### DISCUSSION TITLE
我遇到变量错误

### DISCUSSION THREAD

**Discussion by akayedi** [NewWorldComingSoon/llvm-msvc-issues#41] [Q&A]
我很多变量都是传参过去然后在进行调整，像这种错误我应该如何解决？以下为错误

错误incompatible pointer to integer conversion initializing 'int' with an expression of type 'void *' [-Wint-conversion] 

错误incompatible integer to pointer conversion passing 'ULONG64' (aka 'unsigned long long') to parameter of type 'PVOID' (aka 'void *') [-Wint-conversion] 
图片预览链接：https://i.328888.xyz/2023/03/11/sGFXw.png

![U0{E900RM2VK%$CX5JBV4K3](https://user-images.githubusercontent.com/90334896/224476906-da2c42c5-a9a5-4008-9b5e-d9a9b2bd9e8e.png)

**Comment by akayedi** [comment]
我正常编译是可以通过得，请问有什么好方法可以解决得吗？

**Comment by gmh5225** [comment]
Clang的语法规范比较严格，走的GCC规范，MSVC的比较宽松。这一点上面，强行把error改warning的做法，可以做，但不是特别好。也不打算这样改。
推荐的做法是 取消 "treat warnings as errors" 或者自己做代码层面的强制转换。

**Reply by akayedi** [reply]
我在VS编译器上一直都是设置得《取消》将警告视为错误

**Reply by akayedi** [reply]
![I1FPLIJH G6(9D7X0M@7PHW](https://user-images.githubusercontent.com/90334896/225259639-b0b3f94c-f9e7-435e-9d62-73121715c40f.png)
